lib/fetch/request.js

```diff
@@ -388,7 +393,7 @@ class Request {
         } catch {}
 
         util.addAbortListener(signal, abort)
-        requestFinalizer.register(ac, { signal, abort })
+        requestFinalizer.register(ac, { signal, abort, controller: ac })
```

Comment:
This does not work because it will make the finalizer _nerver_ called, because the `ac` is always referenced. You would need to call `.unregister` elsewhere.

Reply:
I will find a way to resolve this.